### PR TITLE
Search filter styling

### PR DESCRIPTION
**Pull request by asooge**
Screenshot: https://drive.google.com/file/d/15eRwxH_KPBAVVXJv0d4sAz5DjQwBdIeJ/view

- still need to add the magnifying glass image
- sizing is responsive on mobile
- does not always match width of Review component but does for most views.

### CODE REVIEW

File: src/components/App/App.scss [hunk 1]
```diff
@@ -33,7 +33,8 @@
   display: flex;
   position: fixed;
   z-index: 12;
-  left: 6%; top: 4%; right: 6%;
+  left: 20px; top: 20px;
```
Comment (yingsongsana) [inline src/components/App/App.scss:36]: I would do `top: 15px;` that way we won't have to go through and adjust the workspace height/top to match the search bar.

Comment (asooge) [reply src/components/App/App.scss:36]: I wouldn't want to go less than 20px. The wireframes are actually looking for 32px but that would intersect with the workspace so I decided to go with 20px. I don't think we need to change the workspace height/top, but please compare to the deployed site and let me know if any views are not looking good.

**Comment by asooge** [comment]
I added a media query for the tablet view (between 420px and 768px) to match the width of the workspace. Here is an updated screen capture: https://drive.google.com/file/d/1SgytKXi5rnImqk6DRpijLf7hZew-XjFE/view

**Comment by asooge** [comment]
Updated layout to match wireframes. Nav/Search/Filter is 32px from top and 25px from left. Workspace is 45px from the bottom of the search bar. Header is 7px from bottom of the search bar (on Desktop) and on top of the search bar for mobile.

**Comment by yingsongsana** [comment]
The new changes look good. I do wonder if the workspace on mobile view will have even more of a limited view with more margin added to it at the top (with the browser adding to the total height of what's visible.) 

Otherwise, would you mind changing the top for `.userAlertCard` from 15px to 12px? With the search bar lower, those two things are right on top of each other. 
Screen Shot 2020-06-24 at 5 59 53 PM

**Comment by asooge** [comment]
Im not sure how to handle `.userAlertCard`. Right now it is set to `top: 12%;` perhaps we think about it and update `.userAlert` in a separate PR. It would be nice to have it centered on the page with an arrow that points to the POI.

**Comment by yingsongsana** [comment]
Sorry I looked at the wrong css... for tablet, it's set to `top: 15%;`. For laptop, it's set to `top:10%;` It should be changed `top: 12%;` for laptop. 

The reason it's not directly at the center of the page is because it would cover up the POI itself. With each click on any POI, we're re-centering the map with that POI.

**Comment by yingsongsana** [comment]
Oh, I just saw your change now. Disregard the css comment I just mentioned.

**Comment by asooge** [comment]
Okay got it. Also added the magnifying glass: 
image

**Pull request merged**